### PR TITLE
update docker readme and build os version

### PR DESCRIPTION
**Pull request by metaclips**
## Proposed Changes
update docker readme and build os version

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://www.ockam.io/learn/how-to-guides/contributing/CONTRIBUTING#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://www.ockam.io/learn/how-to-guides/high-performance-team/conduct/).
- [x] I have accepted the Ockam [Contributor Licence Agreement](https://www.ockam.io/learn/how-to-guides/contributing/cla/) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/ockam-network/contributors/blob/master/CONTRIBUTORS.csv) file in a separate pull request to the [ockam-network/contributors](https://github.com/ockam-network/contributors) repository.

### CODE REVIEW

**Review by sobi3ch — APPROVED** [review]
LGTM

File: tools/docker/README.md [hunk 1]
```diff
@@ -3,31 +3,44 @@ From the root directory of the ockam codebase:
...
+- This image has things all container images will need - entrypoint, locales, curl.
+- It is based on a lightweight version of [Debian](https://hub.docker.com/_/debian).
+- It has no build tools.
+- It is designed to be the base of images that are running in production and don’t need build tools.
+
+Build the base:
+
 ```
 docker build \
-  --build-arg BASE_IMAGE=debian:10.7-slim@sha256:240f770008bdc538fecc8d3fa7a32a533eac55c14cbc56a9a8a6f7d741b47e33 \
```
Comment (jared-s) [inline tools/docker/README.md:8]: Is this command line option truly not needed? If it's not, can it be omitted from `ockam/builder_base` below too?

Comment (metaclips) [reply tools/docker/README.md:8]: Its updated version is already specified in the [Dockerfile](https://github.com/ockam-network/ockam/blob/5e8af98de33949e057d04848afbca4587ed2f23c/tools/docker/base/Dockerfile#L1), do we still need to pass it as an argument?

Comment (jared-s) [reply tools/docker/README.md:8]: Right, but the image and hash are already specified in an `ARG` statement in the Dockerfile. Just making sure we want the redundancy here.

Comment (metaclips) [reply tools/docker/README.md:8]: Added --build-arg

**Pull request merged**